SP23-P03-Mobile/Config.ts

```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+
+export const Api = axios.create(
+    {
+        baseURL: "https://selu383-sp23-p03-g04.azurewebsites.net/api",
```

Comment:
To keep the URL relative, change the base URL to just "/api".

Axios/Fetch will change the "host" part of the URL to whatever the hostname of the server the application is currently running on is.

This will cause issues with CORS in production because of cross-origin requests